### DISCUSSION TITLE
Welcome to api-user-java Discussions!

### DISCUSSION THREAD

**Discussion by tkubica12** [DevOpsHackathon2021/api-user-java#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tkubica12** [comment]
Testing discussions feature

**Comment by jjindrich** [comment]
Taky si tu klikam co to vlastne umi. Ty tymy, prava atd, to je fakt slaby proti AzureDevOps

**Comment by jjindrich** [comment]
Ke vsemu je tam diskuse v ramci Teamu, oddelena.
